changelog/fix-4990-outdated-package-version

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: dev
+Comment: Only changes a version number in package-lock.json.
```

Comment:
I'm guessing that this won't be included on the release changelog. If it does, it might be more clear to clarify which version number: "Update package-lock.json to use the correct woocommerce-payments version number."

Reply:
I can confirm that it won't be included in the changelog 😀 This comment was added when prompted by the `npm run changelog` command:

>You omitted the changelog entry, which is fine. But please comment as to why no entry is needed.